.gitmodules

```diff
@@ -1335,3 +1335,6 @@
 [submodule "vendor/grammars/zephir-sublime"]
 	path = vendor/grammars/zephir-sublime
 	url = https://github.com/phalcon/zephir-sublime
+[submodule "vendor/grammars/Praat-tmLanguage"]
+	path = vendor/grammars/Praat-tmLanguage
+	url = https://github.com/JJWRoeloffs/Praat-tmLanguage.git
```

Comment:
Why not use the upstream grammar? Linguist has no problems with VSCode grammars as they're TextMate compatible.